lib/datetime/datetime.ex

```diff
@@ -480,6 +484,13 @@ defimpl Timex.Protocol, for: DateTime do
       err
   end
 
+  defp retain_precision(
+         %DateTime{microsecond: {ms, _prevision}} = new_datetime,
```

Comment:
Just noticed a minor typo: `_prevision` should be `_precision` :slightly_smiling_face: